lib/chirp_web/live/post_live/form_component.html.heex

```diff
@@ -0,0 +1,19 @@
+<div>
+  <h2><%= @title %></h2>
+
+  <.form
```

Comment:
@Vretamoza did you fully understand the difference between `leex` and `heex` engines?

Reply:
I know the interpolation sintax changed and the sigil used to write the function component changed from ~L to ~H.